projects/deca/syn/deca.qsf

```diff
@@ -0,0 +1,12 @@
+set_global_assignment -name FAMILY "MAX 10"
+set_global_assignment -name DEVICE 10M50DAF484C6GES
+set_global_assignment -name TOP_LEVEL_ENTITY deca
+set_global_assignment -name VHDL_FILE ../vhdl/DECA.deca/deca_types.vhdl
+set_global_assignment -name VHDL_FILE ../vhdl/DECA.deca/deca.vhdl
+set_global_assignment -name SDC_FILE ../vhdl/DECA.deca/deca.sdc
+set_global_assignment -name QSYS_FILE ../vhdl/DECA.deca/altpllE588ED61A4853C9C.qsys
+set_global_assignment -name VHDL_FILE ../vhdl/DECA.deca/resetSynchronizer.vhdl
+set_global_assignment -name SDC_FILE deca_ext.sdc
+set_global_assignment -name AUTO_RESTART_CONFIGURATION ON
+set_global_assignment -name ENABLE_CONFIGURATION_PINS OFF
+set_global_assignment -name ENABLE_BOOT_SEL_PIN OFF
```

Comment:
I think @DigitalBrains1 made a TCL script reading the manifest json, maybe that could be reused here!

Reply:
Yeah, that would be nice. My plan is to finish the corresponding blog post tomorrow though: explicitly stating that the template will be improved over time. 

I want to release early and often.

Reply:
Jep good plan :). Have fun writing.

Reply:
Alright, got the TCL working for Quartus